SR2019/2018-10-16-sr2019-full.md

```diff
@@ -0,0 +1,11 @@
+---
+to: Student Robotics 2019 Intrest
+subject: The London kickstart is full
+note: This email is to be sent to those who are _not_ in the 'Student Robotics 2019' list.
+---
+
+Hello!
+
+We regret to inform you that Student Robotics 2019 is full. This year we are running on reduced resources, and are unable to host as many teams as we have in previous years.
+
+We invite you to keep an eye on our website, to follow this competition years progress, and we hope to run a larger competition for Student Robotics 2020.
```

Comment:
Possibly say we will contact you when we open spaces next year

Reply:
This is quite important. We don't want to lose their interest

Reply:
I agree, but i'd assume it's a given? Just because they can't enter this competition, doesn't mean they can't enter any competition ever